rfcs/20210510-sum-with-zero-point/README.md

```diff
@@ -0,0 +1,116 @@
+# Proposal for support sum with zero-point
+
+## Motivation
+The motivation to provide such support is to enable dequantize the
+asymmetrically quantized sum's src1 tensor to f32 domain before performing the
+sum operation. In general we want to support following operation:
+`dst[:] := scale * (dst[:] - zero_point) + op(...)`
+
+## Proposal
+The proposal is to extend API by adding the possibility of using such
+functionality. Support for reference solutions should also be added. For
+consistency also need to implement zero points support in standalone sum
+primitive. The solution should also include the benchdnn extension.
+
+## Design
+
+Currently, the sum operation also supports scale. Zero point support should
+be designed to keep backward compatibility. This function is targeting for
+quantization and there is no needed to add support for selection of parameter.
+
+### API extension
+
+1. Overload `append_sum(scale, dt)` by adding next `append_sum(scale, zp, dt)`
+function.
+    - API:
+        ```cpp
+        // include/oneapi/dnnl/dnnl.h
+        dnnl_status_t DNNL_API dnnl_post_ops_append_sum_v3(
+                dnnl_post_ops_t post_ops, float scale, int32_t zero_point,
+                dnnl_data_type_t data_type);
+
+        // include/oneapi/dnnl/dnnl.hpp
+        void append_sum(float scale = 1.f, int32_t zero_point,
```

Comment:
default values are only allowed for trailing arguments
https://en.cppreference.com/w/cpp/language/default_arguments